app-typescript/components/ToggleBox/SimpleToggleBox.tsx

```diff
@@ -106,7 +102,7 @@ export class SimpleToggleBox extends React.PureComponent<IPropsSimple, IState> {
                     {badge ? badge : null}
                 </a>
                 <div className="toggle-box__content-wraper">
-                    <div id={this.htmlId} className={childrenClasses} aria-hidden={!isOpen} ref={this.contentRef}>
+                    <div id={this.htmlId} className={'toggle-box__content' + (this.state.isAnimating && ' toggle-box__content--animation')} aria-hidden={!isOpen} ref={this.contentRef}>
```

Comment:
Didn't I tell you not to use boolean operators when you're not trying to get a boolean result? (except JSX shorthand)

run `'classname1' + (false && ' classname2')`

Reply:
...so:
'toggle-box__content' + (this.state.isAnimating ? ' toggle-box__content--animation' : ' ')?

Reply:
using that classnames function is fine too, I didn't ask not to use it. I was asking not to create a variable

Reply:
ahh i miss that point